EXPERIMENTS.md

```diff
@@ -0,0 +1,188 @@
+# Model checking experiments
+
+**August 2024**
+
+This page summarizes the experiments conducted in the project. While running times may vary between machines, we expect these experiments to be reproducible. More precisely, this means that if you run the experiments with the same parameters, you should obtain the same results.
+
+The above statement about reproducibility comes with a caveat: When using
+randomized symbolic execution, there is still a chance that you might find an
+invariant violation later, unless we support the invariant with an inductive
+proof in another experiment.
+
+If you have any questions about our experimental setup and the experiments, feel
+free to ask [Igor Konnov][].
+
+## 1. Experimental setup
+
+**Hardware**. We conducted the experiments on a dedicated AMD Ryzen 9 5950X
+16-Core Processor with 128GB of RAM. This powerful machine allowed us to run
+multiple experiments in parallel. Apalache is powered by the SMT solver [Z3][],
+which is known to have significant variations in running times. Therefore,
+instead of taking the time figures on this page literally, consider them as
+estimates such as "minutes", "hours", or "days".
+
+**Software**. We are using the symbolic model checker [Apalache][] for
+TLA<sup>+</sup>. Check the Apalache [installation instructions][Apalache
+installation] in the manual. For reproducibility, we recommend using the latest
+release in the 0.45.x series. See [Apalache releases][]. The experiments were
+run on Ubuntu Server 24.04.
+
+In this document, we provide commands for running each experiment in a single
+thread. We also have an experimental setup that runs individual experiments on
+multiple CPU cores. If you are curious about this setup, please contact us.
+
+## 2. Input specifications
+
+<!-- TODO: this section should go into a separate high-level document in the
+future -->
+
+We have produced the following specifications in the project:
+ 
+ - **Spec 1**. This is the specification [spec/ffg_recursive.tla][]. It is the
+ result of a manual mechanical translation of the original executable
+ specification in Python, which can be found in [ffg.py]. This specification is
+ using mutually recursive operators, which are not supported by Apalache.  As a
```

Comment:
I would add a link to https://apalache-mc.org/docs/apalache/principles/recursive.html